app/views/local_authorities/index.html.erb

```diff
@@ -13,7 +13,13 @@
       locals: {placeholder: "Filter list of local authorities"} %>
     </thead>
     <tbody>
-      <%= render @authorities %>
+      <% @authorities.each do |authority| %>
```

Comment:
The previous version of this delegated to another view for rendering each authority object (the view in question is `app/views/local_authorities/_local_authority.html.erb` - the [rails api documentation for partial rendering](http://api.rubyonrails.org/classes/ActionView/PartialRenderer.html) and in particular the bit about ["the default case"](http://api.rubyonrails.org/classes/ActionView/PartialRenderer.html#class-ActionView::PartialRenderer-label-Rendering+the+default+case) has more details). 

I don't have strong opinions on which is better; inlining the loop (what we have now) or using another partial (what we had before), but we should clear up the old partial if we prefer inlining, or go back to using the partial if we prefer using partials.